modules/apps/document-library/document-library-document-conversion/src/main/java/com/liferay/document/library/document/conversion/internal/DocumentConversionImpl.java

```diff
@@ -353,9 +418,13 @@ private void _validate(String targetExtension, String id) {
 	private CompanyLocalService _companyLocalService;
 
 	private DocumentConverter _documentConverter;
+	private volatile NoticeableExecutorService _noticeableExecutorService;
```

Comment:
You're not using anything specific to the noticeable subinterface. Can you use the `ExecutorService` interface and rename the variable to `_executorService`?

Reply:
Thanks because portalExecutorManager.getPortalExecutor returns a NoticeableExecutorService, but I'll change it.